package.json

```diff
@@ -23,37 +23,37 @@
     "node": ">=0.10.0"
   },
   "dependencies": {
-    "URIjs": "1.14.1",
-    "bluebird": "2.3.10",
+    "URIjs": "^1.14.2",
```

Comment:
I think it won't take care of it if someone adds a new (dev)dependency and rewraps it.